relays/bin-substrate/src/chains/wococo_messages_to_rococo.rs

```diff
@@ -226,13 +216,15 @@ pub async fn run(
 			lane.clone(),
 			lane_id,
 			ROCOCO_CHAIN_ID,
+			bp_wococo::WITH_ROCOCO_MESSAGES_PALLET_NAME,
 			params.target_to_source_headers_relay,
 		),
 		RococoTargetClient::new(
 			params.target_client,
 			lane,
 			lane_id,
 			WOCOCO_CHAIN_ID,
+			bp_rococo::WITH_WOCOCO_MESSAGES_PALLET_NAME,
```

Comment:
Couldn't all the constant parameters be part of a single trait, similarly to how this works in the runtime?